src/components/shared/C3Chart/C3Chart.styles.js

```diff
@@ -47,7 +47,7 @@ export const C3ChartStyled = styled.div`
     opacity: 1;
   }
 
-  .c3-shapes-quantity {
```

Comment:
Why this is changed now? Won't it break?

Reply:
because quantity is the name of the BE attribute, change for the generic class